diff --git a/yggdrasil/config.py b/yggdrasil/config.py
index bf2eb3e26..af0ce0f7d 100644
--- a/yggdrasil/config.py
+++ b/yggdrasil/config.py
@@ -341,8 +341,11 @@ def update_language_config(languages=None, skip_warnings=False,
             ygg_cfg_usr.set(idrv.language, 'disable', 'False')
         if ygg_cfg_usr.get(idrv.language, 'disable', 'False').lower() == 'true':
             continue  # pragma: no cover
-        miss += idrv.configure(ygg_cfg_usr,
-                               **lang_kwargs.get(idrv.language, {}))
+        try:
+            miss += idrv.configure(ygg_cfg_usr,
+                                   **lang_kwargs.get(idrv.language, {}))
+        except BaseException:
+            pass
     ygg_cfg_usr.update_file()
     ygg_cfg.reload()
     if not skip_warnings:
diff --git a/yggdrasil/demos/fspm2020 b/yggdrasil/demos/fspm2020
index 0acd22eb2..4abaf302e 160000
--- a/yggdrasil/demos/fspm2020
+++ b/yggdrasil/demos/fspm2020
@@ -1 +1 @@
-Subproject commit 0acd22eb2e811f6180c7438829fa8c5260d186bd
+Subproject commit 4abaf302ef4fd17872ba934e8235c35cbfa61920
diff --git a/yggdrasil/drivers/CModelDriver.py b/yggdrasil/drivers/CModelDriver.py
index 5cae0bf96..cdab30076 100755
--- a/yggdrasil/drivers/CModelDriver.py
+++ b/yggdrasil/drivers/CModelDriver.py
@@ -881,10 +881,10 @@ class CModelDriver(CompiledModelDriver):
             print(f"Setting vcpkg_dir to {vcpkg_dir}"
                   f" ({os.path.abspath(vcpkg_dir)})")
             vcpkg_dir = os.path.abspath(vcpkg_dir)
-            if not os.path.isdir(vcpkg_dir):  # pragma: debug
-                raise ValueError("Path to vcpkg root directory "
-                                 "does not exist: %s." % vcpkg_dir)
-            cfg.set(cls._language, 'vcpkg_dir', vcpkg_dir)
+            if os.path.isdir(vcpkg_dir):  # pragma: debug
+                cfg.set(cls._language, 'vcpkg_dir', vcpkg_dir)
+            else:
+                vcpkg_dir = None
         if cls._language == 'c':
             if macos_sdkroot is None:
                 macos_sdkroot = cfg.get('c', 'macos_sdkroot', _osx_sysroot)
diff --git a/yggdrasil/rapidjson b/yggdrasil/rapidjson
index bfdcf4911..10380a118 160000
--- a/yggdrasil/rapidjson
+++ b/yggdrasil/rapidjson
@@ -1 +1 @@
-Subproject commit bfdcf4911047688fec49014d575433e2e5eb05be
+Subproject commit 10380a118a5619386e7cb4954c4e0353bd009054
